tsMuxer/osdep/textSubtitlesRenderFT.cpp

```diff
@@ -27,7 +27,20 @@ const static char FONT_ROOT[] = "/System/Library/Fonts/";
 #include <freetype/ftstroke.h>
 #include <fs/systemlog.h>
 
+#if !defined(FS_EXP) && !defined(FS_GHC)
 #include <filesystem>
+namespace fs = std::filesystem;
+#endif
+
+#ifdef FS_EXP
+#include <experimental/filesystem>
+namespace fs = std::experimental::filesystem;
+#endif
+
+#ifdef FS_GHC
+#include <ghc/filesystem.hpp>
+namespace fs = ghc::filesystem;
+#endif
```

Comment:
Can be replaced with a little more (IMHO) straightforward if/else tree : 
```suggestion
#if defined(FS_EXP)
#include <experimental/filesystem>
namespace fs = std::experimental::filesystem;
#elif defined(FS_GHC)
#include <ghc/filesystem.hpp>
namespace fs = ghc::filesystem;
#else
#include <filesystem>
namespace fs = std::filesystem;
#endif
```

This also won't break horribly if both `FS_EXP` and `FS_GHC` were defined, for whatever reason.